src/itensor.jl

```diff
@@ -71,11 +71,18 @@ dims(T::ITensor) = dims(inds(T))
 
 dim(T::ITensor) = dim(inds(T))
 
+Base.ndims(A::ITensor) = order(inds(A))
+size(A::ITensor) = dims(inds(A))
+size(A::ITensor, d) = d in 1:ndims(A) ? dim(inds(A)[d]) : 
+  d>0 ? 1 : error("arraysize: dimension out of range")
+
 isNull(T::ITensor) = (store(T) isa Dense{Nothing})
 
 copy(T::ITensor) = ITensor(copy(inds(T)),copy(store(T)))
 
 Array(T::ITensor) = storage_convert(Array,store(T),inds(T))
+Base.Matrix(A::ITensor) = ndims(A) == 2 ? Array(A) : error("expected ndims=2")
```

Comment:
Should also be `ArgumentError` here

Reply:
Is `DimensionMismatch` also OK? If not I can change it.